tests/vmi_lifecycle_test.go

```diff
@@ -292,11 +295,11 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 			}
 
 		},
-			Entry("[test_id:3197]enabled when debugLogs label defined", map[string]string{"debugLogs": "true"}, nil, true),
-			Entry("[test_id:8530]enabled when customLogFilters defined", nil, map[string]string{v1.CustomLibvirtLogFiltersAnnotation: fakeLibvirtLogFilters}, true),
-			Entry("[test_id:8531]enabled when log verbosity is high", map[string]string{"logVerbosity": "10"}, nil, true),
-			Entry("[test_id:8532]disabled when log verbosity is low", map[string]string{"logVerbosity": "2"}, nil, false),
-			Entry("[test_id:8533]disabled when log verbosity, debug logs and customLogFilters are not defined", nil, nil, false),
+			Entry("[test_id:3197]enabled when debugLogs label defined", "debugLogs", "true", "fakeKey", "fakeVal", true),
+			Entry("[test_id:8530]enabled when customLogFilters defined", "fakeKey", "fakeVal", v1.CustomLibvirtLogFiltersAnnotation, fakeLibvirtLogFilters, true),
+			Entry("[test_id:8531]enabled when log verbosity is high", "logVerbosity", "10", "fakeKey", "fakeVal", true),
+			Entry("[test_id:8532]disabled when log verbosity is low", "logVerbosity", "2", "fakeKey", "fakeVal", false),
+			Entry("[test_id:8533]disabled when log verbosity, debug logs and customLogFilters are not defined", "fakeKey", "fakeVal", "fakeKey", "fakeVal", false),
```

Comment:
It bothers me that we're setting unnecessary labels/annotations.

Instead I would advise something like:
```go
options := []Option{libvmi.WithResourceMemory("1Mi")}
if vmiLabelKey != "" {
	options = append(options, libvmi.WithLabel(vmiLabelKey, vmiLabelValue))
}
if vmiAnnotationKey != "" {
	options = append(options, libvmi.WithAnnotation(vmiAnnotationKey, vmiAnnotationValue))
}

vmi := libvmi.New(options...)
```

Reply:
DOne.